lib/fog/vsphere/requests/compute/create_vm.rb

```diff
@@ -106,10 +107,18 @@ def device_change attributes
         def boot_options attributes
           # NOTE: you must be using vsphere_rev 5.0 or greater to set boot_order
           # e.g. Fog::Compute.new(provider: "vsphere", vsphere_rev: "5.5", etc)
-          return unless @vsphere_rev.to_f >= 5
-          RbVmomi::VIM::VirtualMachineBootOptions.new(
-            :bootOrder => boot_order(attributes)
-          )
+          options = {}
+          if @vsphere_rev.to_f >= 5 and attributes[:boot_order]
+            options[:bootOrder] = boot_order(attributes)
+          end
+
+          # Set attributes[:boot_retry] to a delay in miliseconds to enable boot retries
+          if attributes[:boot_retry]
+            options[:bootRetryEnabled] = true
```

Comment:
I can add it to the model, but I was following the pattern of the :boot_order attribute which does not exist on the model.

Neither of them are trivial to add to the model because they don't correspond one-to-one with a property so they can't just go into `Fog::Compute::Vsphere::Shared#ATTR_TO_PROP`.

If it's important for them to be on the model, I can work around this limitation.